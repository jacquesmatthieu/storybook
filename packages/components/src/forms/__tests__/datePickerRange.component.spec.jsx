import React from 'react';
import { shallow } from 'enzyme';

import DatePickerRange from '../datePickerRange.component';

describe('DatePickerRange', () => {
  let props;

  beforeEach(() => {
    props = {};
  });

  const getWrapper = () => shallow(<DatePickerRange {...props} />);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
