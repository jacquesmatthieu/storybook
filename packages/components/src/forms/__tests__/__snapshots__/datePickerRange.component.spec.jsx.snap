// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePickerRange should render properly 1`] = `
<datePickerRangecomponent__StyledWrapperDatePickerRange
  variant="input"
>
  <datePickerRangecomponent__StyledWrapperDatePicker>
    <datePickerRangecomponent__StyledDatePickerInput
      max=""
      onChange={[Function]}
      placeholder=""
      type="date"
      value=""
    />
    <Icon
      fill="grey.main"
      name="slash"
      size={20}
      theme={
        Object {
          "palette": Object {},
        }
      }
    />
    <datePickerRangecomponent__StyledDatePickerInput
      min=""
      onChange={[Function]}
      placeholder=""
      type="date"
      value=""
    />
  </datePickerRangecomponent__StyledWrapperDatePicker>
</datePickerRangecomponent__StyledWrapperDatePickerRange>
`;
